pkg/scripts/os_flatcar.go

```diff
@@ -207,37 +155,6 @@ func RemoveBinariesFlatcar() (string, error) {
 	return result, fail.Runtime(err, "rendering removeBinariesFlatcarScriptTemplate script")
 }
 
-func UpgradeKubeadmAndCNIFlatcar(cluster *kubeoneapi.KubeOneCluster) (string, error) {
-	data := Data{
-		"KUBERNETES_VERSION":     cluster.Versions.Kubernetes,
-		"KUBERNETES_CNI_VERSION": defaultKubernetesCNIVersion,
-		"INSTALL_CONTAINERD":     cluster.ContainerRuntime.Containerd,
-	}
-
-	if err := containerruntime.UpdateDataMap(cluster, data); err != nil {
-		return "", err
-	}
-
-	result, err := Render(upgradeKubeadmAndCNIFlatcarScriptTemplate, data)
-
-	return result, fail.Runtime(err, "rendering upgradeKubeadmAndCNIFlatcarScriptTemplate script")
-}
-
-func UpgradeKubernetesBinariesFlatcar(cluster *kubeoneapi.KubeOneCluster) (string, error) {
-	data := Data{
-		"KUBERNETES_VERSION": cluster.Versions.Kubernetes,
-		"INSTALL_CONTAINERD": cluster.ContainerRuntime.Containerd,
-	}
-
-	if err := containerruntime.UpdateDataMap(cluster, data); err != nil {
-		return "", err
-	}
-
-	result, err := Render(upgradeKubeletAndKubectlFlatcarScriptTemplate, data)
-
-	return result, fail.Runtime(err, "rendering upgradeKubeletAndKubectlFlatcarScriptTemplate script")
-}
-
 const defaultKubernetesCNIVersion = "1.3.0"
```

Comment:
That looks old, we should revisit it at some point soon.

Reply:
It's still recommended version in the [upstream docs](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl)

Reply:
Docs are wrong, it is supposed to be 1.6.0 with v1.32: https://github.com/kubernetes/kubernetes/blob/cdc807a9e849b651fb48c962cc18e25d39ec5edf/build/dependencies.yaml#L19-L30

You can check this file on each branch for the recommended version, this is the actual version that Kubernetes has been tested with.

Reply:
Oh... well, shit. We should fix this.